.github/workflows/main.yml

```
@@ -12,5 +12,5 @@ jobs:
       - name: Comment PR
         uses: thollander/actions-comment-pull-request@master
         with:
-          message: 'This branch can be previewed at [https://explorer.decentraland.zone/branch/${{ github.head_ref }}/index.html](https://explorer.decentraland.zone/branch/${{ github.head_ref }}/index.html?ENV=org)'
+          message: "This branch can be previewed at [https://explorer.decentraland.zone/branch/${{ github.head_ref }}/index.html](https://explorer.decentraland.zone/branch/${{ github.head_ref }}/index.html?ENV=org)"
```

```suggestion
          message: "This branch can be previewed at [https://play.decentraland.zone/branch/${{ github.head_ref }}/index.html](https://play.decentraland.zone/branch/${{ github.head_ref }}/index.html?ENV=org)"
```

`explorer` just redirect to `play`.
For example in unity-renderer we're using `play` instead of `explorer` on the README.md to build the link.

Is just for consistency.